### PR TITLE
 MS: poorly-formatted district addresses #1872

### PR DESCRIPTION
**Pull request by narendra36**
MS: poorly-formatted district addresses #1872 and poorly-formatted author url (was giving 404 error) in bill.py is fixed.

### CODE REVIEW

**Comment by narendra36** [comment]
Please review it @mileswwatkins

File: openstates/mi/people.py [hunk 1]
```diff
@@ -83,8 +83,12 @@ def scrape_lower(self, chamber):
 
     def scrape_upper(self, chamber):
         url = 'http://www.senate.michigan.gov/senatorinfo_list.html'
+        photo_base_url = 'http://www.senate.michigan.gov/senatorinfo_complete.html'
```
Comment (jamesturk) [inline openstates/mi/people.py]: please split MI and MS commits into separate branches

File: openstates/ms/people.py [hunk 2]
```diff
@@ -84,13 +84,27 @@ def scrape_details(self, chamber, leg_name, leg_link, role):
...
             home_city = root.xpath('string(//H_CITY)')
             home_zip = root.xpath('string(//H_ZIP)')
-
-            home_address_total = "%s\n%s\n%s\n%s" % (
-                home_address,
-                home_address2,
-                home_city,
-                home_zip
-            )
+            home_address_total = 'Not Available'
```
Comment (jamesturk) [inline openstates/ms/people.py]: instead of setting to Not Available, just leave it None (i.e. skip the add_contact_detail call)

Comment (narendra36) [reply openstates/ms/people.py]: Ok

**Pull request closed without merging**